### DISCUSSION TITLE
Date Picker 支持自定义时区吗？Does Date Picker support customized timezone?

### DISCUSSION THREAD

**Discussion by helloint** [rsuite/rsuite#3491] [General]
支持local timezone （System timezone）没问题，但能支持指定时区吗？比如我现在在`Asia/Shanghai`，我希望Date Picker显示的是`America/Los_Angeles`（IANA time zone）

**Comment by simonguo** [comment]
如果你是想要显示非本地市区的时间，你试一下用 `date-fns-tz` 在 CustomProvider 中配置 formatDate 来格式时间。

```tsx
import { formatInTimeZone } from "date-fns-tz";
import { DatePicker, CustomProvider } from "rsuite";

const App = () => (
  <CustomProvider
    formatDate={(date, formatStr) => {
      return formatInTimeZone(date, "America/Los_Angeles", formatStr);
    }}
  >
    <DatePicker format="HH:mm:ss"  />
  </CustomProvider>
);
```

**Reply by coolyard** [reply]
使用CustomProvider虽然可以保证最终input里显示的时间 是按照指定timezone的，但是在Date Picker里选择日期时间的时候还是System timezone，对用户来说还是有问题
<img width="544" alt="image" src="https://github.com/rsuite/rsuite/assets/24653185/27bb637e-17a0-42b7-9033-c09cb4d3b33a">

**Reply by coolyard** [reply]
Demo:
https://codesandbox.io/p/sandbox/rsuite-datepicker-customprovider-format-timezone-fnv6rr
